test/basic.js

```diff
@@ -182,6 +182,18 @@ t.test('GLOBSTAR marker exposed', t => {
 
 t.test('globstar re matches zero or more path portions', t => {
   const cases = {
+    'path/**': {
+      'path/x.html': true,
+      'path/x/y.html': true,
+      'path/x/y/z.html': true,
+      'path//x.html': true,
+      'path//x//y.html': true,
+      'path//x//y//z.html': true,
+      'path.html': false,
```

Comment:
Without the fix, this test-case (and the cases below) do match with `makeRe`, similar to what has been described in the original issue: 'some/path/**' matching 'some/path-but-different'